src/test/incremental/hashes/if_expressions.rs

```diff
@@ -94,7 +94,7 @@ pub fn add_else_branch(x: bool) -> u32 {
 }
 
 #[cfg(not(cfail1))]
-#[rustc_clean(cfg="cfail2", except="HirBody,typeck_tables_of")]
```

Comment:
What causes this regression? I don't think it should happen. Is it the `_` pattern requiring a type?

Reply:
Don't mind me... I misread what's going on.